src/sage/combinat/permutation.py

```diff
@@ -5339,18 +5342,18 @@ def merging_cycles(list_of_cycles):
                     gamma = Permutation(tuple(new_Perm))
                     yield gamma
 
-        def rewind(L, k):
+        def rewind(L, n):
             """
-            Construct the list M such that M[(j*k)%(len(M))] == L[j].
+            Construct the list M such that M[(j*n)%(len(M))] == L[j].
```

Comment:
Maybe this docstring could be rewritten without code, but this is really minor.  If not, also enclose it in double single quotes, and stick to the spacing rules.